packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -0,0 +1,574 @@
+pragma solidity ^0.5.3;
```

Comment:
No changes to `ReleaseGold`, just changed the name from `ReleaseGoldInstance` to `ReleaseGold` since `instance` is redundant